### PR TITLE
_peerdb_uid: be uuid

### PR DESCRIPTION
**Pull request by serprex**

### CODE REVIEW

**Comment by heavycrystal** [comment]
is there a reason we're doing this or is this just to look cleaner?

File: flow/connectors/postgres/postgres.go [hunk 1]
```diff
@@ -451,7 +451,7 @@ func syncRecordsCore[Items model.Items](
 				}
 
 				row = []any{
-					uuid.New().String(),
+					uuid.New(),
```
Comment (heavycrystal) [inline flow/connectors/postgres/postgres.go:454]: should test if switching this with the old raw table syntax will break syncs
ideally we'd have a fallback or a migration

File: flow/connectors/utils/stream.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ func recordToQRecordOrError[Items model.Items](batchID int64, record model.Recor
 		return nil, fmt.Errorf("unknown record type: %T", typedRecord)
 	}
 
-	entries[0] = qvalue.QValueString{Val: uuid.New().String()}
+	entries[0] = qvalue.QValueUUID{Val: uuid.New()}
```
Comment (heavycrystal) [inline flow/connectors/utils/stream.go:124]: same here

**Comment by serprex** [comment]
> is there a reason we're doing this or is this just to look cleaner?

it also fits in storage cleaner

File: flow/connectors/postgres/postgres.go (hunk 1 above)
Comment (serprex) [reply flow/connectors/postgres/postgres.go:454]: tested with old schema, postgres can handle inserting uuid into text column

**Comment by iskakaushik** [comment]
@serprex existing mirrors in CH will continue to work after this and doesn't require any special migration i'm assuming?

**Review by heavycrystal — APPROVED** [review]

File: flow/connectors/utils/stream.go (hunk 2 above)
Comment (heavycrystal) [reply flow/connectors/utils/stream.go:124]: SF and BQ as well, unless we're not upgrading customers using them

**Comment by serprex** [comment]
> @serprex existing mirrors in CH will continue to work after this and doesn't require any special migration i'm assuming?

correct

**Pull request merged**